View/AptLayout.py

```diff
@@ -1,88 +1,116 @@
 import tkinter as tk
 from PIL import Image, ImageTk, ImageEnhance
+from model.abstract_event_observer import EventObserver
+from model.observer import Observer
+import model.demo_model as demo_model
 import os
 
-
-class AptLayout:
-    def __init__(self, parent):
-
-        # Darkness intensity
-        self.darknessIntensity = 0.4
+class AptLayout(EventObserver):
+    def __init__(self, parent, model : demo_model):
+        super().__init__()
```

Comment:
Is this super init needed since it is an abstract class?

Reply:
No probably not